### PR TITLE
Add python frontend

### PR DESCRIPTION
**Pull request by waynetee**
#26

This is my first GitHub contribution, please let me know if there are any issues.

### CODE REVIEW

**Comment by lama0206** [comment]
Thank you for your effort. Looks good to me after quick glance. I don't have a computer with me right now and take a closer look later.

It seems that you have added the new plugin twice to the dependency management though (https://github.com/waynetee/jplag/blob/7883c3b82a16602c11323a641977978d3532dc5c/jplag.parent/pom.xml#L224). Can you please remove one of the entries?

**Comment by waynetee** [comment]
Thanks for the prompt reply, it's fixed.

**Comment by waynetee** [comment]
Hi, have you had the chance to look at it?

**Comment by waynetee** [comment]
Added some minor enhancements

File: jplag.frontend.scheme/src/main/java/jplag/scheme/Language.java [hunk 1]
```diff
@@ -21,7 +21,7 @@ public int errorsCount() {
 									   // Konstruktor
 
 	public String[] suffixes() {
-		String[] res = { ".scm", ".SCM" };
+		String[] res = { ".scm", ".SCM", "ss" };
```
Comment (erikmd) [inline jplag.frontend.scheme/src/main/java/jplag/scheme/Language.java]: Hi @waynetee, shouldn't this be ".ss" (with a dot)?

**Comment by waynetee** [comment]
Whoops, I forgot to mention its fixed

**Comment by lama0206** [comment]
Sorry that it took so long. Thanks for your effort!

**Pull request merged**